src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/ILanguageServerExtensions.cs

```diff
@@ -1,20 +1,18 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System.Reflection;
 using System.Threading;
 using System.Threading.Tasks;
-using OmniSharp.Extensions.LanguageServer.Server;
+using OmniSharp.Extensions.LanguageServer.Protocol.Server;
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer
 {
     public static class ILanguageServerExtensions
     {
         public static Task InitializedAsync(this ILanguageServer languageServer, CancellationToken cancellationToken)
         {
-            var type = typeof(OmniSharp.Extensions.LanguageServer.Server.LanguageServer);
-            var method = type.GetMethod("Initialize", BindingFlags.NonPublic | BindingFlags.Instance);
-            var task = (Task)method.Invoke(languageServer, new object[] { cancellationToken });
+            var ls = languageServer as OmniSharp.Extensions.LanguageServer.Server.LanguageServer;
+            var task = ls.Initialize(cancellationToken);
```

Comment:
I wonder if we actually still need to do this. I'd suggest trying to comment this out and see if things work

Reply:
The cast is probably not required anymore, I added the `Initialize` to the interface iirc.